### PR TITLE
Remove prop-types from react build

### LINKED ISSUE #225: Remove prop-types from react build
[Prop-types is used](https://github.com/bem/bem-react-core/blob/master/src/presets/React.js#L2) in react build. Using `bem-react-core` in react project without `prop-types` causes errors (`prop-types` is peerDependency). Seems like we can simple remove it

### PR DESCRIPTION
**Pull request by awinogradov**
Close #225

cc @Vittly

### CODE REVIEW

File: src/Bem.js [hunk 1]
```diff
@@ -172,11 +172,11 @@ export default function({ preset, naming }) {
         displayName : 'Bem',
 
         childContextTypes : {
-            bemBlock : PropTypes.string
+            bemBlock : () => null
         },
 
         contextTypes : {
-            bemBlock : PropTypes.string
+            bemBlock : () => null
```
Comment (Vittly) [inline src/Bem.js]: `const PropTypesStub = () => null;` in the beginning will be more obvious )

**Comment by Vittly** [comment]
thank you!

File: src/Bem.js (hunk 1 above)
Comment (awinogradov) [reply src/Bem.js]: Maybe...

Comment (veged) [reply src/Bem.js]: my vote for inline comment in code, without unnecessary constant

**Pull request merged**